### DISCUSSION TITLE
GDAL not gonna install

### DISCUSSION THREAD

**Discussion by mikey694201989** [community/community#135159] [Programming Help] [answered]
### Body

pip instal gdal is not working and it said that i have to install wheel im a little bit confused could you help?

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by aradfarahani** [comment, marked as answer]
To install OSgeo for Python, you'll need to follow these steps:

1. **Install GDAL**: OSgeo is often used as a placeholder for GDAL. You should install GDAL directly using pip:
   ```bash
   pip install gdal
   ```

2. **Install Wheel**: If you encounter issues with wheel files, you need to ensure that the `wheel` package is installed. You can do this by running:
   ```bash
   pip install wheel
   ```

3. **Install the Package**: If you have a `.whl` file for the package, you can install it using pip. First, navigate to the directory where the `.whl` file is located, then run:
   ```bash
   pip install <your-package>.whl
   ```

4. **Verify Installation**: After installation, you can verify it by trying to import the module in a Python shell:
   ```python
   from osgeo import gdal
   ```

If you encounter any issues, make sure your Python environment is correctly set up and that you have the necessary permissions to install packages.


thanks to https://www.lfd.uci.edu/

**Reply by mikey694201989** [reply]
Thanks a lot.
